CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `opentelemetry-instrumentation`: Avoid calls to `context.detach` with `None` token.
   ([#3673](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3673))
 
+### Added
+
+- `opentelemetry-instrumentation`: teach opentelemetry-instrument about gevent monkeypatching
```

Comment:
I think this would be easier to grep, and other entries are similar:

```suggestion
- `opentelemetry-instrumentation`: add support for  `OTEL_PYTHON_AUTO_INSTRUMENTATION_EXPERIMENTAL_GEVENT_PATCH` to inform opentelemetry-instrument about gevent monkeypatching
```